src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -12,44 +12,47 @@
 import com.google.codeu.data.User;
 import org.jsoup.Jsoup;
 import org.jsoup.safety.Whitelist;
+import org.commonmark.node.*;
```

Comment:
suggestion:  you should avoid wildcard imports in order (1) not to bloat the application, (2) to let the reader know exactly what classes you're using.

See https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports